static/js/zamboni/buttons.js

```diff
@@ -179,16 +178,28 @@ var installButton = function() {
         };
     });
 
+    var showDownloadAnyway = function($button) {
+        var $installShell = $button.parents('.install-shell');
+        var $downloadAnyway = $installShell.find('.download-anyway');
+        if ($downloadAnyway.length) {
+            var href = $button.filter(':visible').attr('href');
+            var pos = href.lastIndexOf('/');
+            href = escape_(href.slice(0, pos) + '/type:attachment/' + href.slice(pos + 1));
+            var $newParent = $installShell.find('.more-versions');
+            $newParent.append(' | ');
+            $newParent.append($downloadAnyway);
+            $downloadAnyway.attr('href', href);
+            $downloadAnyway.show();
+        }
+    }
```

Comment:
Can you maybe post a screenshot how this is supposed to look for QA to check as well?